framework/test/unit/modules/interoperability/internal_method.spec.ts

```diff
@@ -1622,23 +1878,31 @@ describe('Base interoperability internal method', () => {
 				.spyOn(regularMerkleTree, 'calculateRootFromRightWitness')
 				.mockReturnValue(channelData.partnerChainOutboxRoot);
 
+			const params = {
+				...crossChainUpdateParams,
+				inboxUpdate: {
+					crossChainMessages: [],
+					messageWitnessHashes: [],
```

Comment:
a more specific value, e.g. one entry with a random value, would be good to ensure that the last argument for `calculateRootFromRightWitness` was indeed the right one.

Reply:
updated [here](https://github.com/LiskHQ/lisk-sdk/blob/4cbdadaf90ceccfe5fdf242f688ee3d30da90c4d/framework/test/unit/modules/interoperability/internal_method.spec.ts#L1900C11-L1900C11)